WORKSPACE

```diff
@@ -18,12 +18,6 @@ nodejs_register_toolchains(
     node_version = "16.9.0",
 )
 
-load("@aspect_bazel_lib//lib:repositories.bzl", "DEFAULT_YQ_VERSION", "register_yq_toolchains")
-
-register_yq_toolchains(
```

Comment:
after a release here, we'll get to simplify the e2e/workspace/WORKSPACE file for all the downstream repos too :)